src/less/day-slot.less

```diff
@@ -30,20 +31,25 @@
     min-height: 1em;
   }
 
+  .rbc-time-slot {
+    border-top: 1px solid lighten(@cell-border, 10%);
+  }
 }
 
 .rbc-time-slot {
   flex: 1 0 0;
-  min-height: 20px;
-  height: 20px;
 
-  & + & {
-    border-top: 1px solid @cell-border;
+  &.rbc-now {
+    font-weight: bold;
   }
+}
 
-  &:nth-child(even){
-    border-top-color: lighten(@cell-border, 10%);
-  }
+.rbc-timeslice-group {
+  border-bottom: 1px solid @cell-border;
+  min-height: 40px;
+  height: 40px;
```

Comment:
I think I'm being unclear. :)

If you haven't already, run npm run storybook and browse to localhost:9002 
then check out the examples with 3 and 9 slots.

Then click week view, and select as much as you can. The default selectable 
view allows you to make a slot of at most about 15.5 hours, depending on 
screen size. The 3 slots one allows selecting about 23 hours, the 9 slots 
one allows selecting the whole thing with about half the page representing 
24 hours.

The key thing to remember is that the step is preserved, the only thing 
that changes is the visual representation. I will use this to have 5 minute 
step and still visualize 9 to 6 in a single page.


via CloudMagic Email 
[https://cloudmagic.com/k/d/mailapp?ct=pa&cv=8.5.38&pv=5.1.1&source=email_footer_2]